requirements.in

```diff
@@ -1,11 +1,10 @@
-aiohttp
 aiokatcp>=1.5.0
 dask
 katsdpservices[aiomonitor]>=1.2
 katsdpsigproc[CUDA]>=1.6.1
 katsdptelstate
 numpy
-prometheus-async
+prometheus-async[aiohttp]
```

Comment:
This now matches what's in `setup.cfg`.

I'm tempted to have a look into `pip-compile` to see if it can be tweaked to use `setup.cfg` as the input, rather than a separate `requirements.in`, that'll be one less file to maintain.

Reply:
From the web site it sounds like it supports it pretty easily. I guess the trick will be checking whether it will play nice with pre-commit.